.github/workflows/frontend_format.yaml

```diff
@@ -0,0 +1,18 @@
+name: frontend_format
+
+on:
+  push:
```

Comment:
Not running it on pushes to other branches primarily as it'd probably be a waste. I think you would want this? 
```suggestion
  push: [main, dev]
  pull_request:
    types: [opened, reopened]
```

Reply:
Why would it be a waste? I think all branches should have the code that is pushed formatted according to the standards set

Reply:
No one is noticing branches being worked on and pushed until they hit up a PR. And you are usually running a prettier task on save. So it's just an unnecessary waste of ci minutes. If you don't feel like so, though, sure, what's the better config? @mmxz-mmxz

Reply:
I just thought that it might be better to have each push correctly formatted to avoid nitpicks in code review or side effects from weird formatting when the branch is merged to main. However this may cause issues if the ci does make formatting changes and the user doesn't pull the changes afterwards. I'm kinda mixed on this as I usually use husky for formatting, but we can just keep original config `push: [main, dev]` as `[opened, reopened]` won't run on subsequent commits which still doesn't solve the issue, and adding `synchronize` would be the same as `[ push ]`.

Reply:
That makes sense.